### ISSUE TITLE
Feature Request: Share Link available on MiniPlayer or Tray PopUp

### ISSUE THREAD

**Issue by pewsey** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2623]
I run with the MiniPlayer in a corner and can hover the tray popup. However, to share a song link, I need to open the player.

It would enhance the experience to be able to generate that URL to the clipboard, then I can trivially paste it as required.

It's also entirely possible I've not yet found the right place on the MiniPlayer to be able to get the share link.

**Comment by tekfranz** [comment]
Yes, if we could see the"More Options" Menu in the Mini Player it would be great!
The Three Vertical Dots should not take much space. The Share Option Add to Library, Add to Playlist would then open from the Mini Player.

_Also of interest would be the Thumbs Up and Thumbs Down but there is probably no space in the mini Player for those without getting creative._
